.github/actions/semantic-version/action.yml

```diff
@@ -0,0 +1,27 @@
+name: "Semantic version"
```

Comment:
update `build-dotnet-app.yml` file within this PR so that we can test this action in practice